src/app/hooks/useProposalList.ts

```diff
@@ -1,7 +1,18 @@
 import { useEffect, useState } from 'react';
-import { network } from '../containers/BlockChainProvider/network';
 import { ContractName } from '../containers/BlockChainProvider/types';
 import { Proposal } from '../../types/Proposal';
+import { aggregate } from '@makerdao/multicall';
+import { isMainnet } from 'utils/helpers';
+
+const configTestnet = {
+  rpcUrl: 'https://public-node.testnet.rsk.co',
+  multicallAddress: '0x9e469e1fc7fb4c5d17897b68eaf1afc9df39f103',
+};
+
+const configMainnet = {
+  rpcUrl: 'https://public-node.rsk.co',
+  multicallAddress: '0x6c62bf5440de2cb157205b15c424bceb5c3368f5',
+};
```

Comment:
Use rpc nodes from classifiers file, add multicall contracts to contract.ts file and use getContract() helper to get it for current chain